OsuPlayer.Tests/OsuPlayer.IO/PlaylistTests.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OsuPlayer.Data.OsuPlayer.Classes;
+using OsuPlayer.IO.Storage.Playlists;
+
+namespace OsuPlayer.Tests;
+
+public class PlaylistTests
+{
+    [Test]
+    public async Task SetCurrentPlaylist()
+    {
+        PlaylistManager.SetCurrentPlaylist(default);
+        PlaylistManager.SetCurrentPlaylist(new ());
+        
+        await PlaylistManager.SetCurrentPlaylistAsync(default);
+        await PlaylistManager.SetCurrentPlaylistAsync(new ());
+    }
```

Comment:
Please split synced/async Tests as they are different cases. Does apply for every test case with both calls.

Reply:
Done